### ISSUE TITLE
ffmpeg dependency

### ISSUE THREAD

**Issue by mynameisvinn** [thomas-haslwanter/scikit-sound#1]
how do you feel about factoring out the ffmpeg dependency?

**Comment by thomas-haslwanter** [comment]
I have mixed feelings.
Currently I try to do the following:
- Try to find out if ffmpeg is installed
- If I cannot find it, ask the user if they have it; and store that information, so it can be accessed later on

That overhead is a one-time question (on each system, in the worst case).

If we leave that away, we would either be able to only process WAV-files; or have to do the process described above anyway. So the only benefit would be for uses who only work with WAV-files, and have no idea what they are doing. My current thinking is that most users will want the option to be able to work with at least MP3-files; and the simplest approach that I have found was the installation of ffmpeg.

So if things are up to me, I would leave them as they are. But I would be more than interested in other thoughts about the issue, and about the best possible approach.
